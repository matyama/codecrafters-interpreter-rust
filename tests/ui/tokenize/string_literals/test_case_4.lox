"foo"+"hello""perseverance" && "Success" != "Failure"
