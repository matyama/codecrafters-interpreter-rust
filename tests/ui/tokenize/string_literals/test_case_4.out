STRING "foo" foo
PLUS + null
STRING "hello" hello
STRING "perseverance" perseverance
[line 1] Error: Unexpected character: &
[line 1] Error: Unexpected character: &
STRING "Success" Success
BANG_EQUAL != null
STRING "Failure" Failure
EOF  null
